app/main.ipynb

```diff
@@ -192,18 +236,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "A.mean(axis=1)",
```

Comment:
The mean calculation should be done along the `axis=0` to find the mean for each column, not `axis=1`. Change to `A.mean(axis=0)`.